assets/dev/js/frontend/utils/assets-loader.js

```diff
@@ -78,7 +78,6 @@ AssetsLoader.assets = {
 		},
 	},
 	style: {
-		// TODO: Remove 'swiper' in v3.28.0 [ED-16258].
```

Comment:
We can't remove this, because the swiper styling is used in every current lightbox.